manufacturers/prob_match_manufacturers.R

```diff
@@ -0,0 +1,186 @@
+# Keying the drug company data
+# S. Kirmer
+# Feb 2017
+# Data4Democracy
+
+library(dplyr)
+library(data.table)
+library(reshape2)
+library(sem)
+library(doBy)
+library(ggplot2)
+library(AER)
+library(ivpack)
+library(readxl)
+library(RecordLinkage)
+library(stringr)
+library(gdata)
+library(stringdist)
+library(stats)
+
+companies_drugs <- read.csv("U://Medicaid_Drug/drugdata_clean.csv", stringsAsFactors = FALSE)
```

Comment:
I'm not an expert in R, so I'll defer to others here; however, IMHO we should try to use relative file paths if we can so that people can run scripts without hitting environment-specific errors. That is to say, this code only works if you have the data files in a `U://Medicaid_Drug` directory.

Again, not sure how best to handle this, but my Pythonic approach would be to download from data.world via code into a local directory (maybe even the current directory) and open the downloaded file with `read.csv()`.